### PR TITLE
Html css

### PR DESCRIPTION
**Pull request by DharmaSenareddy-Zemosolabs**

### CODE REVIEW

File: aboutme.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>About Me</title>
+    <style>
+      body {
```
Comment (nagavinay-zemoso) [inline aboutme.html:6]: instead of style tag, you can have css files and import them

File: mygallery.html [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+          <li><a href="contact.html">Contact</a></li>
+          <li><a href="mygallery.html">My Gallery</a></li>
+        </ul>
+      </nav>
+    </header>
+    
+    <h1>My Gallery</h1>
+    <div class="gallery-container">
+      <div class="photo-thumbnail">
+        <img src="298633121_170633715530467_5899080429385164709_n.jpg" alt="Photo 1">
```
Comment (nagavinay-zemoso) [inline mygallery.html:72]: maintain a proper naming for images, and have a proper folder structure like images, all the imahes should be in images folder and same for the css too

**Comment by nagavinay-zemoso** [comment]
maintain proper naming conventions while giving commit messages and branch names, and add description like what changes you have made in that pr while raising a pr

**Comment by nagavinay-zemoso** [comment]
navigation is not working properly, plz check it

File: aboutme.html (hunk 1 above)
Comment (DharmaSenareddy-Zemosolabs) [reply aboutme.html]: Done

File: mygallery.html (hunk 2 above)
Comment (DharmaSenareddy-Zemosolabs) [reply mygallery.html]: Done

**Pull request closed without merging**